server.js

```diff
@@ -86,21 +97,30 @@ var createAndLogEvent = function(data, req) {
     var value = parseValue(properties[key]);
     entry += ' ' + key + '=' + value;
   }
-  entry += '\n';
-  fs.appendFile(path.resolve(__dirname, './events.log'), entry, function(err) {
-    if (err) {
-      console.log(err);
-    } else {
-      //console.log("Logged tracked data");
-    }
-  });
+  if (argv.docker) {
+    console.info(entry);
+  } else {
+    entry += '\n';
+    fs.appendFile(path.resolve(__dirname, './events.log'), entry, function(err) {
+      if (err) {
+        console.error(err);
+      } else if (argv.debug) {
+        console.log('Logged tracked data');
+      }
+    });
+  }
 };
 
 /*
  * Use Middlewares
  **********************************
  */
-app.use(express.logger());
+if (argv.docker) {
+  var morgan = require('morgan');
+  app.use(morgan('combined', { stream: process.stderr }));
```

Comment:
Perfect! Sorry, I'm not at all familiar with the module you were using so I didn't catch that.. Looks good then.